.github/workflows/tests.yml

```diff
@@ -29,13 +29,16 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -e .[dev]
+    - name: Build Images
+      run: |
+        # these tags need to match the ones in tests/gpu-tests/test-local.yaml
+        docker build -t igitman/nemo-skills:0.7.0 -f dockerfiles/Dockerfile.nemo-skills .
```

Comment:
could we only do this if certain files are changed to avoid rebuilding everytime unnecessarily? I think it's only needed if either requirements, dockerfile or sandbox file changes

Reply:
Not fully sure I understand this. If main gets merged to those, they will get these changes, right?